skops/card/_model_card.py

```diff
@@ -28,6 +28,19 @@ def wrap_as_details(text: str, folded: bool) -> str:
     return f"<details>\n<summary> Click to expand </summary>\n\n{text}\n\n</details>"
 
 
+def _clean_table(table: str) -> str:
+    # replace line breaks "\n" with html tag <br />, however, leave end-of-line
+    # line breaks (eol_lb) intact
+    eol_lb = "|\n"
+    placeholder = "$%!?"  # arbitrary sting that never appears naturally
```

Comment:
as a friend once said, equivalent to banging my head against the keyboard lol

Reply:
It's stupid :D If you have a better idea, please let me know

About the PR number: I fixed that. How do you add the correct number before you even created the PR? Do you just check the highest number so far and hope that until you actually open the PR, there hasn't been another PR in-between?

Reply:
I don't, I just add that as a commit after I create the PR :D